### DISCUSSION TITLE
no netcdf file created

### DISCUSSION THREAD

**Discussion by norafried** [Parcels-code/Parcels#1270] [Q&A]
I'm running particles tracks on the POP data. I did some shorter test runs which all worked fine and now performed a five year run where I released around 4.3 Million particle. The script was killed after all trajectories were done (I could see 100%) and no netcdf file was created. All the trajectories are now saved in this temporary folder as .npy! 
 what could be the reason for no NetCDF file? Can I do something to still create one from the particle tracks? I tried the function "parcels_convert_npydir_to_netcdf" but I don't seem to have that function. Not sure if relevant but I'm using Parcles 2.5.1!
 
 Thanks for your help.

**Comment by erikvansebille** [comment]
As of v2.4.0 the output of Parcels is not in netcdf anymore, but in zarr-format. That means that the conversion script also does not exist anymore. See https://github.com/OceanParcels/parcels/releases/tag/v2.4.0 and links in that message

**Reply by norafried** [reply]
sorry, I made a typo. I'm using version 2.1.5. and I don't get .zarr files. Is there no way to convert them anymore?

**Reply by erikvansebille** [reply]
This issue with missing netcdf files and difficulty converting the .npy files is exactly why we moved to zarr format in v2.4.0. Why do you run with cut an old version? I recommend you updating to the latest version (which also fixes a lot of other bugs!)

**Reply by norafried** [reply]
For loading in the data to POP I was in contact with @pdnooteboom and we were only able to load in the data correctly with an older version. So this is the reason why I use a rather old version.
Is there a reason for not creating a netCDF? I'm running it remotely and I cannot say why it broke down. Could it be a memory error that the netcdf file get's too big? I never had this issue when trying with shorter runs.
